### PR TITLE
[Dev] add contributor to CI authority

### PR DESCRIPTION
**Pull request by hutiechuan**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Comment by Rhett-Ying** [comment]
@dgl-bot

File: Jenkinsfile [hunk 1]
```diff
@@ -157,7 +157,7 @@ def is_authorized(name) {
     'rudongyu', 'classicsong', 'HuXiangkun', 'hetong007', 'kylasa',
     'frozenbugs', 'peizhou001', 'zheng-da', 'czkkkkkk', 'thvasilo',
     // Intern:
-    'keli-wen', 'caojy1998', 'RamonZhou', 'xiangyuzhi', 'Skeleton003', 'yxy235',
+    'keli-wen', 'caojy1998', 'RamonZhou', 'xiangyuzhi', 'Skeleton003', 'yxy235','hutiechuan'
```
Comment (Rhett-Ying) [inline Jenkinsfile:160]: add it into next line to avoid this line to be too long.

Comment (Rhett-Ying) [reply Jenkinsfile:160]: and `,` is missing

**Review by Rhett-Ying — COMMENTED** [review]
please always name commit like `[dev] add contributor to CI authority`

File: Jenkinsfile (hunk 1 above)
Comment (hutiechuan) [reply Jenkinsfile:160]: ok

**Comment by Rhett-Ying** [comment]
@dgl-bot

**Review by Rhett-Ying — APPROVED** [review]

**Pull request merged**